app.py

```diff
@@ -2059,9 +2059,12 @@ def get_all_users_with_all_info():
     return users
 
 
-def format_entry(val, other = ''):
+def format_entry(val, other=''):
+    val.strip()
     if val == 'Other':
         return f"\"Other: {other}\""
+    elif val.startswith("\"") and val.endswith("\""):
```

Comment:
@libpitt it's not sufficient to just check the double quotes at the beginning and the end. The double quotes can appear within the string value, including those "other" fields with user provided data. We should replace all instances. As a result of escaping the double quotes, it'll break in the case where the original string value already contains escape characters. So you'll need to escape that first to treat `\"` as `"`.

It's better to handle the above escape at the beginning of the `format_entry(val, other='')` function for `val`  when it's not "Other", then do the same escape for `other` value within that `if val == 'Other':` block.

Reply:
Hi @yuanzhou Can you please provide a full edge case of where this would  break? 

Earlier you gave an example to format in the form: `Other: "user provided value"`
But this would actually does not work and breaks the csv.

Thanks.